src/web/browser/App.tsx

```diff
@@ -42,11 +44,23 @@ export const hydrateApp = ({ CAPI, NAV }: { CAPI: CAPIType; NAV: NavType }) => {
 
 const App = ({ CAPI, NAV }: Props) => {
     const [isSignedIn, setIsSignedIn] = useState<boolean>(false);
+    const [countryCode, setCountryCode] = useState<string>('');
 
     useEffect(() => {
         setIsSignedIn(!!getCookie('GU_U'));
     }, []);
 
+    useEffect(() => {
+        const callFetch = async () => {
+            setCountryCode(await getCountryCode());
+        };
+        callFetch();
+    }, []);
+
+    console.log(
```

Comment:
console.log

Reply:
That one was intentional! I wanted to give Andre a pointer but also keep typescript happy (no unused vars...)